runtime/trappist/src/lib.rs

```diff
@@ -213,17 +214,21 @@ impl pallet_authorship::Config for Runtime {
 }
 
 impl pallet_balances::Config for Runtime {
-	type MaxLocks = ConstU32<50>;
 	/// The type for recording an account's balance.
 	type Balance = Balance;
+	type DustRemoval = ();
 	/// The ubiquitous event type.
 	type RuntimeEvent = RuntimeEvent;
-	type DustRemoval = ();
 	type ExistentialDeposit = ConstU128<EXISTENTIAL_DEPOSIT>;
 	type AccountStore = System;
-	type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
+	type WeightInfo = weights::pallet_balances::WeightInfo<Runtime>;
+	type MaxLocks = ConstU32<50>;
 	type MaxReserves = ConstU32<50>;
 	type ReserveIdentifier = [u8; 8];
+	type HoldIdentifier = ();
+	type FreezeIdentifier = ();
+	type MaxHolds = ConstU32<0>;
```

Comment:
Is it okey that we keep this values at `ConstU32<0>`. Not completly sure what would be the best alternative

Reply:
I think currently `construct_runtime` macro generates `RuntimeFreezeReason` and `RuntimeHoldReason` enums based on `#[pallet::composite_enum]` attributes in pallets. These should be used as ***Identifiers here. 
As freezes are the new locks and reserves were deprecated in favor of holds I think the corresponding limits can be reused. 
See:
- https://github.com/paritytech/substrate/pull/12951
- https://github.com/paritytech/substrate/blob/master/frame/support/procedural/src/construct_runtime/expand/hold_reason.rs
- https://github.com/paritytech/substrate/blob/master/frame/support/procedural/src/construct_runtime/expand/freeze_reason.rs

But anyway this is not critical for the migration and can be addressed in a separate pr.